### ISSUE TITLE
filter-slivers remove-empty bug

### ISSUE THREAD

**Issue by 2803media** [mbloch/mapshaper#644]
Hi

with this command 

```
mapshaper communes.json \
    -clip remove-slivers monu.json \
    -filter-slivers min-area=0.05km2 remove-empty \
    -clean \
    -drop geometry \
    -filter-fields code \
    -o monu.csv format=csv
```

I get this error:

`Error: [filter-slivers] Received one or more unexpected parameters: remove-empty`

 Any idea?

**Comment by mbloch** [comment]
Hi! thanks for reporting this bug... I haven't had much time to work on mapshaper recently, but I'll try to get this fixed soon.

**Comment by mbloch** [comment]
This should be fixed in v0.6.98. Thanks for reporting!
